cloud/deployment/deployment.go

```diff
@@ -241,31 +257,32 @@ func Create(label, workspaceID, description, clusterID, runtimeVersion, dagDeplo
 		}
 	}
 
+	var isCicdEnforced bool
+	if cicdEnforcement == "" {
+		isCicdEnforced = false
+	}
+	if cicdEnforcement == disable {
+		isCicdEnforced = false
+	}
+	if cicdEnforcement == enable {
+		isCicdEnforced = true
+	}
```

Comment:
```suggestion
	isCicdEnforced := false
	if cicdEnforcement == enable {
		isCicdEnforced = true
	}
```

nit we can simplify to this